planner-api/openapi.yaml

```diff
@@ -169,6 +169,37 @@ paths:
         "404":
           description: "Goal with that ID does not exist"
 
+  "/api/goals/{id}/subGoals":
```

Comment:
I used hyphens in my endpoint (/api/auth/access-token), instead of camelCase. So for sake of consistency, either change subGoals to sub-goals or access-token to accessToken. Same problem on line 192.